libusb/version_nano.h

```diff
@@ -1 +1 @@
-#define LIBUSB_NANO 11792
```

Comment:
The nano jumps by +2 here.

Reply:
Opps. Forgot to do one more `git amend --no-edit` to fix that. Need to figure out why the scripts do this.